### PR TITLE
Feature/added pause media on prayer time action

### PR DESCRIPTION
**Pull request by mbaraa**
media will pause when a prayer time is up.

### CODE REVIEW

**Comment by mbaraa** [comment]
Moved the media pause logic to a separate class and called from for convenience `com.prayercompanion.prayercompanionandroid.data.receivers.AlramReceiver`

File: app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+){
+    private val audioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
+
+    fun pauseMedia() {
+        val focusRequest: AudioFocusRequest =
+            AudioFocusRequest.Builder(AudioManager.AUDIOFOCUS_GAIN)
+                .setAudioAttributes(
+                    AudioAttributes.Builder().setUsage(AudioAttributes.USAGE_MEDIA).build()
+                )
+                .build()
```
Comment (RaedGhazal) [inline app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt:23]: can this be initialized outside of the function so it's not created each time the function is called?

File: app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.prayercompanion.prayercompanionandroid.data.utils
```
Comment (RaedGhazal) [inline app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt:1]: this class fits more in the domain layer package, can you move it to `prayercompanionandroid.domain.utils`

**Review by RaedGhazal — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt (hunk 2 above)
Comment (mbaraa) [reply app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt]: resolved in [9d51ebe](https://github.com/Prayer-Companion/PrayerCompanionAndroid/pull/19/commits/9d51ebe66ae0172d935f9658bb85aa752c034953)

File: app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt (hunk 1 above)
Comment (mbaraa) [reply app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt]: resolved in [9d51ebe](https://github.com/Prayer-Companion/PrayerCompanionAndroid/pull/19/commits/9d51ebe66ae0172d935f9658bb85aa752c034953)

**Review by RaedGhazal — APPROVED** [review]

**Comment by RaedGhazal** [comment]
Looks good, thanks for your contribution!
I'll test it on my device and will merge if everything seems working.

File: app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/ScheduleDailyPrayersWorker.kt [hunk 3]
```diff
@@ -1,6 +1,9 @@
 package com.prayercompanion.prayercompanionandroid.data.utils
```
Comment (RaedGhazal) [inline app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/ScheduleDailyPrayersWorker.kt:1]: this file's changes are not needed, can you remove them?

**Review by RaedGhazal — CHANGES_REQUESTED** [review]

**Pull request merged**